### PR TITLE
CB-1- Explanation

### PR DESCRIPTION
**Pull request by alekhyapinnamaneni**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
This PR creates a new feature. It adds a Jumbotron component that explains what this project does/how to use it.

* **What is the current behavior? What does the code do before you made the new changes?** 
Before I made the new changes, the code generates a blank page.

* **What is the new behavior?**
After the changes I have made, the code generates a page titled "Recipe Maker" along with an explanation of what the Recipe Maker does. The theme has also been updated to match the dark green/cream theme we chose on the poll on Slack.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No.

* **Other information**:
n/a

### CODE REVIEW

**Review by sunghyun96 — COMMENTED** [review]
Looks good Alekhya! The only thing I can think of is maybe adding some styling to the component later on to blend the design with the other components once we get a better idea of how some of these other components will look.

**Review by sunghyun96 — APPROVED** [review]
I am going to go ahead and give an approval to accompany my comment I made yesterday, since the present code works and any style changes can be made during later sprints if necessary!

**Review by AdemOdza — APPROVED** [review]
Looks good! I agree with Sung, styling could be added later on when we figure out what we want our style to be.

File: frontend/src/components/LandingPage/Explanation/Explanation.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Jumbotron } from 'reactstrap';
+
+const Explanation = (props) =>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Explanation/Explanation.js]: Can we change this into a functional component instead of a const here.

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Alekhya! Left one comment that we can discuss. It also looks like there are some merge conflicts that I can help you resolve if youd like.

**Review by Stacey-Masi — APPROVED** [review]
Nice job finding the color scheme and implementing it. Making the title bold or white might look good as well, but this is great!

**Review by Codubee — APPROVED** [review]

**Pull request merged**